plugins/woocommerce/src/Admin/WCAdminHelper.php

```diff
@@ -230,7 +230,14 @@ public static function is_store_page( $url = '' ) {
 			}
 		}
 
-		return false;
+		/**
+		 * Filter if a URL is a store page.
+		 *
+		 * @since 9.2.0
+		 * @param bool   $is_store_page Whether or not the URL is a store page.
+		 * @param string $url           URL to check.
+		 */
+		return apply_filters( 'woocommerce_is_store_page', false, $url );
```

Comment:
```suggestion
		/**
		 * Filter if a URL is a store page.
		 *
		 * @since 9.3.0
		 * @param bool   $is_store_page Whether or not the URL is a store page.
		 * @param string $url           URL to check.
		 */
		$is_store_page = apply_filters( 'woocommerce_is_store_page', false, $url );
		
		return filter_var( $is_store_page, FILTER_VALIDATE_BOOL );
```

We're trying to get away from returning filtered values without validating that they are still the expected type after filtering. This way, we can guarantee that this function will still always return a boolean. Furthermore, `FILTER_VALIDATE_BOOL` makes it so that most non-boolean values will result in `false` instead of `true`.

(Also updated the `@since` because 9.2 is already in code freeze, so this wouldn't get included until 9.3.)